Client/Frontend/Widgets/TwoLineCell.swift

```diff
@@ -62,6 +65,34 @@ class TwoLineTableViewCell: UITableViewCell {
     }
 }
 
+class HistoryTableViewCell: TwoLineTableViewCell {
+    let borderView = UIView()
+    let iconImageView = UIImageView()
+
+    override var imageView: UIImageView? {
+        return iconImageView
+    }
+
+    override init(style: UITableViewCellStyle, reuseIdentifier: String?) {
+        super.init(style: UITableViewCellStyle.Subtitle, reuseIdentifier: reuseIdentifier)
+        contentView.addSubview(borderView)
+        borderView.addSubview(iconImageView)
+        separatorInset = UIEdgeInsetsMake(0, TwoLineCellUX.BorderWidth + 2 * TwoLineCellUX.BorderViewMargin, 0, 0)
+
+        twoLineHelper.setUpViews(self, textLabel: textLabel!, detailTextLabel: detailTextLabel!, imageView: iconImageView, borderView: borderView)
+    }
+
+    override func prepareForReuse() {
+        super.prepareForReuse()
+        separatorInset = UIEdgeInsetsMake(0, TwoLineCellUX.BorderWidth + 2 * TwoLineCellUX.BorderViewMargin, 0, 0)
+        twoLineHelper.setupHistoryPanelDynamicFonts()
```

Comment:
Why do you need to duplicate these in `prepareForReuse`? The inset/fonts are set in `init...they won't change again, will they?